styles.css

```diff
@@ -451,3 +443,286 @@ form {
+  }
+
+  .title h1 {
+    margin-left: 0;
+    padding: 0;
+  }
+
+  #first_page h1 {
+    margin-left: 260px;
+    padding-top: 281px;
+    margin-bottom: 12px;
+  }
+
+  .upper-part p {
+    margin-left: 0;
+    margin-top: 12px;
+    max-width: 447px;
+  }
+
+  .contact p {
+    margin-left: 436px;
+    margin-right: 434px;
+    margin-top: 12px;
+    margin-bottom: 12px;
+  }
+
+  #first_page p {
+    margin-left: 260px;
+    margin-bottom: 12px;
+  }
+
+  #first_page .connect {
+    margin-left: 260px;
+  }
+
+  #first_page .links ul {
+    margin-left: 260px;
+    padding: 0;
+    padding-bottom: 178px;
+  }
+
+  /* page two */
+
+  #mid {
+    display: grid;
+    grid-template-columns: 142px 1fr 142px;
+  }
+
+  .left-block {
+    grid-column: 2;
+    flex-direction: row;
+  }
+
+  #grid-eight {
+    align-items: left;
+  }
+
+  #second-page .left {
+    margin-top: 48px;
+    margin-left: 46px;
+  }
+
+  #grid-two .primary-text {
+    margin-left: 0;
+    padding-left: 0;
+  }
+
+  #grid-two .tags {
+    margin-left: 0;
+  }
+
+  .frame-2 {
+    margin: 0;
+    padding-bottom: 0 0 0 20px;
+  }
+
+  #grid-two p {
+    margin-left: 46px;
+    max-width: 419px;
+  }
+
+  .left .action {
+    margin-left: 0;
+  }
+
+  .for-small-size {
+    display: none;
+  }
+
+  /* page three */
+
+  #third-page {
+    flex-direction: row-reverse;
+  }
+
+  .right {
+    flex: 1;
+  }
+
+  #third-page h1 {
+    margin: 48px 46px 20px 48px;
+  }
+
+  #third-page .frame-2 {
+    margin: 0;
+    padding: 0;
+  }
+
+  #third-page p {
+    margin: 0 auto 32px 48px;
+    max-width: 498px;
+  }
+
+  .frame-2 .action {
+    margin-left: 60px;
+  }
+
+  #third-page .action {
+    margin: 0 auto 32px 48px;
+  }
+
+  /* page four */
+
+  #fourth-page .left {
+    padding-top: 48px;
+    padding-left: 46px;
+  }
+
+  .left .for-small-size {
+    display: none;
+  }
+
+  #grid-six p {
+    padding-bottom: 26px;
+  }
+
+  /* page five */
+
+  #page-five {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  #page-five .right {
+    margin-left: 48px;
+    margin-top: 48px;
+  }
+
+  #page-five p {
+    padding-bottom: 26px;
+  }
+
+  .right .for-small-size {
+    display: none;
+  }
+
+  /* About page */
+
+  .about-block {
+    display: flex;
+    flex-direction: row;
+    margin-left: 142px;
+    margin-right: 142px;
+  }
+
+  .upper-part .resume {
+    margin-bottom: 142px;
+  }
+
+  .lower-part {
+    margin-top: 166px;
+    margin-left: 49px;
+  }
+
+  .languages .lang {
+    padding-right: 500px;
+  }
+
+  .frame-work-name {
+    padding-right: 500px;
+  }
+
+  .after-frame-works {
+    margin-top: 23px;
+  }
+
+  .skills {
+    margin-bottom: 23px;
+    margin-top: 24px;
+  }
+
+  .skill-name {
+    padding-right: 500px;
+  }
+
+  .after-skills {
+    margin-bottom: 166px;
+  }
+
+  .contact {
+    background-image: url(images/contact\ form-\ background-shape-desktop.svg);
+    padding-left: 142px;
+    padding-right: 142px;
```

Comment:
- [ ] I think it will be better if you reduce the padding or even remove the padding of your contact section to match your chosen Figma design 👍🏽 . Kindly follow the design specification on Figma 👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/194632168-f243d570-788d-46c7-8920-e8f121e09d27.png)  | ![image](https://user-images.githubusercontent.com/63473584/194632707-a7ced9c8-81ea-4dc7-b3d0-21fa9af5489c.png) |